droidmediacamera2.cpp

```diff
@@ -19,13 +19,13 @@
 #include "droidmediacamera.h"
 
 #include <camera/CameraParameters.h>
+#include <camera/NdkCaptureRequest.h>
```

Comment:
I think these were sorted alphabetically before

Reply:
There is an issue with includes in Android versions < 10. In my wip branch I have a comment like this before this include "// This needs to be first because of broken includes in Android < 10"